server/conn_stmt.go

```diff
@@ -471,7 +477,14 @@ func parseBinaryDuration(pos int, paramValues []byte, isNegative uint8) (int, st
 	pos++
 	seconds := uint8(paramValues[pos])
 	pos++
-	return pos, fmt.Sprintf("%s%d %02d:%02d:%02d", sign, days, hours, minutes, seconds)
+
+	var str string
+	if days == 0 {
+		str = fmt.Sprintf("%02d:%02d:%02d", hours, minutes, seconds)
```

Comment:
we do not need to consider the `sign` here?

Reply:
I'll try it.

Reply:
```
java.text.ParseException: Unparseable date: "-0 10:00:00"
```

It is illegal to set the argument like above.